tests/drivers/base_test_metadata.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/env python
+# Copyright 2016 Criteo
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import print_function
+
+import re
+
+from biggraphite import accessor as bg_accessor
+from biggraphite import accessor_cache as bg_accessor_cache
+from biggraphite import test_utils as bg_test_utils
+from biggraphite import glob_utils as bg_glob_utils
+from biggraphite.drivers import cassandra as bg_cassandra
+from biggraphite.drivers import elasticsearch as bg_elasticsearch
+
+
+class BaseTestAccessorMetadata(object):
+
+    def test_glob_metrics(self):
+        IS_CASSANDRA_LUCENE = self.ACCESSOR_SETTINGS.get('cassandra_use_lucene', False)
```

Comment:
Having common tests depending the implementation can break easily once you add a new driver. Should we add a fixme or todo ?